Dockerfile

```diff
@@ -11,7 +11,7 @@ RUN apk add --no-cache curl
 COPY blackfire blackfire-agent /usr/local/bin/
 
 # Don't run as root
-RUN addgroup -S -g 999 blackfire && adduser -S -G blackfire -u 999 blackfire
+RUN addgroup -S blackfire && adduser -S -G blackfire -u 999 blackfire
```

Comment:
You forgot the `-H` option to not create the `/home/blackfire` dir

Reply:
Sorry, fixed